client/src/pages/LandingPage/LandingPage.tsx

```diff
@@ -0,0 +1,16 @@
+import Grid from '@material-ui/core/Grid';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import useStyles from './useStyles';
+import Layout from '../../components/Layout/Layout';
+
+export default function LandingPage(): JSX.Element {
+  const classes = useStyles();
+  return (
+    <Layout>
+      <CssBaseline />
+      <Grid container component="main" className={`${classes.root}`}>
+        <h1>Landing page</h1>
+      </Grid>
+    </Layout>
```

Comment:
Instead of repeating this wrapper code, see if you can refactor so that this (and similar) component only has unique JSX